api/metrics/src/main/java/io/opentelemetry/api/metrics/LongCounterBuilder.java

```diff
@@ -41,6 +41,14 @@
    *
    * <p>The callback will only be called when the {@link Meter} is being observed.
    *
+   * <p>Callbacks are expected to abide by the following restrictions:
+   *
+   * <ul>
+   *   <li>Run in a finite amount of time.
+   *   <li>Safe to call repeatedly, across multiple threads.
+   *   <li>Return positive, monotonically increasing values.
```

Comment:
Not related to the documentation, but should the SDK be defensive against the callbacks throwing, and wrap them in a try / catch?

Reply:
Yes, can you open a bug for that?

Reply:
Issue #3891.